code/builders/builder-vite/src/plugins/webpack-stats-plugin.ts

```diff
@@ -58,7 +59,7 @@ export function pluginWebpackStats({ workingDir }: WebpackStatsPluginOptions): W
     else {
       const relativePath = path.relative(workingDir, stripQueryParams(filename));
```

Comment:
Couldn't we just use `path.posix.relative` here to use forward slashes, independently of the OS?

Reply:
@valentinpalkovic that is how I originally submitted this PR, until @shilman suggested using `slash` to be consistent with how windows paths are already converted elsewhere. I don't mind either way, if you're able to decide between you which option is preferable?

Reply:
I see. Sorry that I haven’t followed the whole conversation! It’s fine then how it is!